pym/bob/input.py

```diff
@@ -1950,6 +1959,12 @@ def prepare(self, pathFormatter, inputEnv, sandboxEnabled, inputStates, inputSan
             varPrivate[key] = env.substitute(value, "privateEnvironment::"+key)
         env.update(varPrivate)
 
+        # apply meta variables
+        for (key, value) in self.__varMeta.items():
+            if key in env:
+                raise ParseError("Duplikate variable in metaVars and environment: " + key)
```

Comment:
Do you think this is necessary? Always giving precedence to meta variables looks viable too...